### PR TITLE
Improved: code to hide option to set baseURL if value available in '.env' file (#29wgkkh).

### PR DESCRIPTION
**Pull request by Mayank909**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/receiving#contribution-guideline)

### CODE REVIEW

File: src/api/index.ts [hunk 1]
```diff
@@ -70,8 +70,12 @@ const api = async (customConfig: any) => {
         params: customConfig.params
     }
     
-    const baseURL = store.getters['user/getInstanceUrl'];
-    if (baseURL) config.baseURL = `https://${baseURL}.hotwax.io/api/`;
+    let baseURL = process.env.VUE_APP_BASE_URL;
+    if(!baseURL){
+        baseURL = store.getters['user/getInstaceUrl'];
```
Comment (adityasharma7) [inline src/api/index.ts:75]:
```suggestion
        baseURL = store.getters['user/getInstanceUrl'];
```

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Login.vue [hunk 2]
```diff
@@ -69,7 +70,8 @@ export default defineComponent({
   },
   methods: {
     login: function () {
-      this.store.dispatch("user/setUserInstanceUrl", this.instanceUrl.trim())
+     if(!this.baseURL) this.store.dispatch("user/setUserInstanceUrl", this.instanceUrl.trim())
```
Comment (adityasharma7) [inline src/views/Login.vue:73]: Indentation issues

Comment (Mayank909) [reply src/views/Login.vue]: Done Sir

File: src/api/index.ts (hunk 1 above)
Comment (Mayank909) [reply src/api/index.ts]: Done Sir

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**